envoy/config/filter/http/jwt_auth/v2/config.proto

```diff
@@ -0,0 +1,188 @@
+// Copyright 2017 Istio Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto3";
+
+package Envoy.Http.JwtAuth.Config;
+
+import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+
+// JSON Web Token (JWT) token format for authentication as defined by
+// https://tools.ietf.org/html/rfc7519. See [OAuth
+// 2.0](https://tools.ietf.org/html/rfc6749) and [OIDC
+// 1.0](http://openid.net/connect) for how this is used in the whole
+// authentication flow.
+//
+// Example,
+//
+//     issuer: https://example.com
```

Comment:
Have you look at the docs output to see how this YAML renders? Should you use RST YAML directives? https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#code-block-directive

Reply:
Try to follow it. Not sure how to view the generated doc output